### PR TITLE
Dynamic Overture versioning in queries

### PR DESCRIPTION
**Pull request by jenningsanderson**
Introduces a new global variable in the configuration: `overtureRelease`.

All SQL queries live in the `src/queries/` directory and have a placeholder `__OVERTURE_RELEASE` in them.

When the how-to pages build, the latest overture release is injected.

Additionally, validation software should be able to crawl the `src/queries` directory and likewise inject the new overture release version for testing.  If any query fails, it can be reported and updated in one place.

https://dfhx9f55j8eg5.cloudfront.net/how-to/pr/9

### CODE REVIEW

**Comment by varapmsft** [comment]
@jamesdaus-msft could you confirm if you will be able to read these queries in your validation test, assuming that we are able to create an artifact from this repo with these queries and deploy to the DataBricks cluster? + @hshamsie  as FYI.

File: src/queries/duckdb/countries_from_azure.sql
Comment (jamesdaus-msft) [inline src/queries/duckdb/countries_from_azure.sql:1]: I believe this query requires the azure extension: https://duckdb.org/docs/extensions/azure.html
And import via LOAD azure;
And potentially setting a connection string, though I am unable to get it running locally right now

**Comment by jamesdaus-msft** [comment]
> @jamesdaus-msft could you confirm if you will be able to read these queries in your validation test, assuming that we are able to create an artifact from this repo with these queries and deploy to the DataBricks cluster? + @hshamsie as FYI.

I believe I should be able to parse the queries as strings and pass them to the validation libraries 👍

File: src/queries/duckdb/countries_from_azure.sql
Comment (jenningsanderson) [reply src/queries/duckdb/countries_from_azure.sql:1]: Yes, it totally does, apologies, I didn't copy the right thing here. Fixing it.

**Review by jamesdaus-msft — APPROVED** [review]

**Pull request merged**